mina.netty/src/main/java/org/kaazing/mina/netty/socket/nio/NioDatagramChannelIoAcceptor.java

```diff
@@ -15,46 +15,29 @@
  */
 package org.kaazing.mina.netty.socket.nio;
 
-import java.net.InetSocketAddress;
-
 import org.apache.mina.core.service.DefaultTransportMetadata;
 import org.apache.mina.core.service.TransportMetadata;
 import org.apache.mina.transport.socket.DatagramSessionConfig;
 import org.jboss.netty.channel.Channel;
 import org.jboss.netty.channel.ChannelConfig;
-import org.jboss.netty.channel.ChannelHandlerContext;
-import org.jboss.netty.channel.ChannelPipeline;
-import org.jboss.netty.channel.ChannelStateEvent;
-import org.jboss.netty.channel.Channels;
-import org.jboss.netty.channel.ChildChannelStateEvent;
 import org.jboss.netty.channel.SimpleChannelUpstreamHandler;
-import org.jboss.netty.channel.socket.Worker;
 import org.jboss.netty.channel.socket.nio.NioDatagramChannel;
 import org.jboss.netty.channel.socket.nio.NioDatagramChannelFactory;
-
 import org.kaazing.mina.core.service.IoProcessorEx;
 import org.kaazing.mina.netty.ChannelIoSession;
 import org.kaazing.mina.netty.socket.DatagramChannelIoAcceptor;
 import org.kaazing.mina.netty.socket.DatagramChannelIoSessionConfig;
 
+import java.net.InetSocketAddress;
+
 public class NioDatagramChannelIoAcceptor extends DatagramChannelIoAcceptor {
 
     private static final TransportMetadata NIO_DATAGRAM_TRANSPORT_METADATA = new DefaultTransportMetadata(
             "Kaazing", "NioDatagramChannel", true, true, InetSocketAddress.class,
             DatagramSessionConfig.class, Object.class);
 
     public NioDatagramChannelIoAcceptor(DatagramChannelIoSessionConfig sessionConfig) {
-        super(sessionConfig, new NioDatagramChannelFactory(), new SimpleChannelUpstreamHandler() {
-            @Override
-            public void childChannelOpen(ChannelHandlerContext ctx, ChildChannelStateEvent e) throws Exception {
-                super.childChannelOpen(ctx, e);
-
-                NioDatagramChannel childChannel = (NioDatagramChannel) e.getChildChannel();
-                Worker childWorker = childChannel.getWorker();
-                ChannelPipeline childPipeline = childChannel.getPipeline();
-                childPipeline.addFirst("mina-alignment", new ChildAlignmentChannelHandler(childWorker));
-            }
-        });
+        super(sessionConfig, new NioDatagramChannelFactory(), new SimpleChannelUpstreamHandler());
```

Comment:
Just using the worker pool created by netty (but for tcp, we do create a woker pool and boss pool and pass it to the factory)